airbyte-workers/src/main/java/io/airbyte/workers/config/ContainerOrchestratorConfigBeanFactory.java

```diff
@@ -37,6 +38,12 @@ public class ContainerOrchestratorConfigBeanFactory {
   private static final String DD_VERSION_ENV_VAR = "DD_VERSION";
   private static final String JAVA_OPTS_ENV_VAR = "JAVA_OPTS";
   private static final String PUBLISH_METRICS_ENV_VAR = "PUBLISH_METRICS";
```

Comment:
Curious @colesnodgrass how come we don't reference a central code file for these names? e.g. the EnvConfigs file